src/ParallelTempering.jl

```diff
@@ -542,3 +614,23 @@ function run_PT!(
 
     return :SUCCESS
 end
+
+# Run PT using kT schedule passed as Vector instead of function
+function run_PT!(
+    replica::Replica, 
+    kT_sched::Vector{Float64}; 
+    therm_mcs::Int64=1000, 
+    measure_interval::Int64=10, 
+    rex_interval::Int64=1, 
+    max_mcs::Int64=1_000_000, 
+    bin_size::Float64=1.0, 
+    print_hist::Bool=false, 
+    print_xyz_ranks::Vector{Int64}=Int64[]
+)
+    kT_func = (i, N)->kT_sched[i]
+
+    return run_PT!(
+        replica, kT_func; 
+        therm_mcs, measure_interval, rex_interval, max_mcs, bin_size, print_hist, print_xyz_ranks
+    )
+end
```

Comment:
This syntax actually does work for passing the kwargs (and you won't have to update this function when the original method changes):

```julia
function run_PT!(replica::Replica, kT_sched::Vector{Float64}; kwargs...)
    kT_func = (i, N) -> kT_sched[i]
    return run_PT!(replica, kT_func; kwargs...)
end

```

Reply:
Oh, gotcha. Good to know